chainermn/__init__.py

```diff
@@ -20,4 +20,4 @@
 
 global_except_hook._add_hook_if_enabled()
 
-__version__ = pkg_resources.get_distribution('chainermn').version
+__version__ = pkg_resources.get_distribution('chainer').version
```

Comment:
Please use `chainer.__version__` instead. (c.f. #3628)

Reply:
I already found the mistake and fixed it! thanks!

Reply:
Ah, I meant `pkg_resources.get_distribution ` should be avoided. It's totally fine to do this in separate PR.

Reply:
I did it in https://github.com/chainer/chainer/pull/5298